pallets/anchors/src/tests.rs

```diff
@@ -295,6 +295,22 @@ fn basic_commit() {
 				message: Some("AnchorStoreDateInPast"),
 			})
 		);
+
+		// commit anchor triggering days overflow
+		assert_noop!(
+			Anchors::commit(
+				Origin::signed(1),
+				pre_image2,
+				doc_root,
+				<Test as frame_system::Config>::Hashing::hash_of(&0),
+				371085174374358017 // triggers overflow
+			),
+			DispatchError::Module(ModuleError {
```

Comment:
Could we use the direct error type as suggested by https://github.com/centrifuge/centrifuge-chain/issues/919?